docs/schedule/validate.ja.md

```diff
@@ -0,0 +1,45 @@
+# GTFSスケジュール検証ツール
+
+このフリーでオープンソースの正規のGTFSスケジュール検証ツールは、GTFSコミュニティの協力を得て、[MobilityDataによって](https://mobilitydata.org)メンテナンスされています。このツールを使って、あなたのデータセットを公式の[GTFSリファレンスと](reference.md) [ベストプラクティスに照らして](best-practices.md)評価することができます。使いやすい検証レポートをHTML形式で提供し、ブラウザで開いて他の関係者と共有することができます。
+
+<!-- <img class="center" src="../../assets/validator_animation.gif" width="150"> -->
+
+<br/>
+
+<div class="usage-buttons">
+   <a class="button" href="https://share.mobilitydata.org/validator-installer-mac">Mac 用ダウンロード<img class="icon" src="../../assets/apple.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/validator-installer-windows">Windows 用ダウンロード<img class="icon" src="../../assets/windows.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/validator-installer-ubuntu">Ubuntu 用ダウンロード</a><a class="button" href="https://share.mobilitydata.org/validator-installer-ubuntu"> <img class="icon" src="../../assets/ubuntu.svg" width="20"/>
+</a></div>
+
+**VERSION v4.0.0**/[リリースノートを](https://github.com/MobilityData/gtfs-validator/releases/latest)読む
+
+<hr>
+
+## 使用方法
+
+<div class="usage">
+    <div class="usage-list">
+        <ol>
+            <li>私たちのオペレーティングシステムに基づいて、プロジェクトの最新バージョンをダウンロードしてください。</li>
+            <li>インストール後、アプリケーションを実行すると、ウィンドウが表示されます。</li>
+            <li>GTFSデータセットを読み込む: URL、ZIPファイル、またはフィードの個々の.txtファイルを含むディレクトリを指定することができます。</li>
+            <li>出力ディレクトリを選択します。これは、検証レポートが書き込まれるディレクトリです。</li>
+            <li>検証が終了すると、結果は自動的にブラウザで開かれます。もし表示されない場合は、出力パスに作成された report.html ファイルを開いてください。</li>
+            <li>バリデータが問題を発見した場合、その問題を修正するためのドキュメントへのリンクが表示されます。検証レポートはHTMLファイルとして保存されますので、簡単に保存し、他の人と共有することができます。</li>
+        </ol>
+    </div>
+    <div class="usage-video">
+        <video class="center" width="560" height="315" controls>
+            <source src="../../assets/validator_demo_large.mp4" type="video/mp4">
+        </video>
+    </div>
+</div>
+
+## データパイプラインの一部としてこのバリデータを使用する
+
+このツールをデータパイプラインで使用する方法の詳細については、[GitHub リポジトリを](https://github.com/MobilityData/gtfs-validator)参照してください。
+
+## 貢献する
+
+このプロジェクトはコミュニティの活動であり、このプロジェクトに興味を持つ人は誰でもコミュニティに参加し、意思決定に参加し、様々な方法でプロジェクトの推進に貢献することができます。新しい機能についてのアイデアを共有する、問題を報告する、ドキュメントを改善する、使い方を教えるなど、さまざまな方法で貢献することができます。
+
+このプロジェクトに貢献することに興味がありますか？[GitHub](https://github.com/MobilityData/gtfs-validator)リポジトリにアクセスし、[貢献のガイドラインを](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md)お読みください。
```

Comment:
or this one since they are both meant to be "Github repository" translated but are different